cms/static/js/certificates/models/certificate.js

```diff
@@ -20,9 +20,25 @@ function (_, str, Backbone, BackboneRelational, BackboneAssociations, gettext, C
         defaults: {
             name: 'Name of the certificate',
             description: 'Description of the certificate',
-            course_title: 'Title of the course',
+
+            /**
+             * It represents the course title.
+             */
+            course_title: '',
+
+            /**
+             * It represents the path to organization logo image.
+             */
             org_logo_path: '',
+
+            /**
+             * It represents the default version of the certificate.
+             */
             version: 1,
+
+            /**
+             * It represents the active state of the certificate. default state is false.
```

Comment:
@catong -- this field is the flag that is set when the user clicks the Activate/Deactivate button in the UI